src/client/index.js

```diff
@@ -2,43 +2,34 @@
 import React from "react"
 import ReactDOM from "react-dom"
 import Router, { browserHistory } from "react-router"
-import { Provider } from "react-redux"
+import { Provider as ReduxContextProvider } from "react-redux"
 
-import fetchJSON from "../fetchJSON"
-import MetadataProvider from "../MetadataProvider"
+import StatinamicContextProvider from "../ContextProvider"
 
 export default function statinamic({
+  layouts,
   metadata,
   routes,
   store,
 }) {
   let devtools = false
-  if (__DEVTOOLS__) {
+  if (process.env.REDUX_DEVTOOLS) {
```

Comment:
The thing is: we need a test that can be statically converted when uglifying. I will add a process.env.CLIENT then...